target/linux/ath79/dts/qca955x_senao_router-dual.dtsi

```diff
@@ -123,4 +131,7 @@
 
 &wmac {
 	status = "okay";
+
+	nvmem-cells = <&calibration_art_1000>, <&macaddr_uboot_eth 2>;
```

Comment:
In my opinion, this is device-specific enough to be put into each dts instead of the dtsi. If I imagine myself as someone who was looking at a dts for the very first time I would be very confused how it knows where the calibration data is with seemingly no reference to it, or if someone had almost identical device but needed to change that one detail it wouldn't be as clear what needed to be done. That way it would be less concealed to the average user who needs to learn dts regardless of experience in order to configure a device.

it doesn't really save much lines anyway...

Reply:
Hmm? Other dtsi/dts combos here do this already as well as https://github.com/openwrt/openwrt/pull/15714

I'd argue the ath10k node should be moved out of the dtsi.

Reply:
Yeah, I am aware that we already do this in many places, I'm just stating that I don't agree with the philosophy of "if there is anything that might be common with another device, move it to a DTSI". I think that whatever we decide either stays in a DTS or is moved to a DTSI should be more categorical.

However, this is not a strong opinion, it's just my thoughts. I think about how some people who may be beginners might have a hard time finding a node if it *only* exists in a DTSI (without an appending block in a DTS) or they might not even realize that more nodes exist if they don't see some representation of it in the DTS.

Reply:
I've reverted moving the wmac node.

Reply:
ah I see that now.... but just a head's up, moving the node is still mentioned in the commit message